lib/OpenQA/CacheService/Response.pm

```diff
@@ -5,8 +5,15 @@ package OpenQA::CacheService::Response;
 use Mojo::Base -base, -signatures;
 
 has [qw(data error)];
+
+# define soft-limit for inactive Minion jobs (enforced when determining idle worker slots availability)
 has max_inactive_jobs => $ENV{OPENQA_CACHE_MAX_INACTIVE_JOBS} // 5;
 
+# define hard-limit for inactive Minion jobs (enforced on setup of already started openQA job)
+has max_inactive_jobs_hard_limit => sub ($self) {
+    $ENV{OPENQA_CACHE_MAX_INACTIVE_JOBS_HARD_LIMIT} // ($self->max_inactive_jobs + 40);
```

Comment:
I would find it more intuitive to default the hard limit 50 rather than depending on the soft limit. And it should be documented accordingly

Reply:
But this way one can increase `OPENQA_CACHE_MAX_INACTIVE_JOBS` and then `OPENQA_CACHE_MAX_INACTIVE_JOBS_HARD_LIMIT` will increase accordingly automatically (which is probably wanted). Not sure whether it makes sense to document is somewhere at this point.